packages/card/src/Card.ts

```diff
@@ -41,12 +41,12 @@ import detailStyles from '@spectrum-web-components/styles/detail.js';
  * @element sp-card
  *
  * @fires change - Announces a change in the `selected` property of a card
- * @slot preview - This is the preview image for Gallery Cards
- * @slot cover-photo - This is the cover photo for Default and Quiet Cards
+ * @slot preview - This is the preview image for Gallery Cards and Quiet Cards
```

Comment:
According to the current source code, `quiet` cards use the `preview` slot not the `cover-photo` slot.